style.css

```diff
@@ -481,3 +494,191 @@ header {
   background-color: #6070ff;
   color: #fff;
 }
+
+@media screen and (min-width: 768px) {
```

Comment:
- [ ] You are expected to maintain the mobile design while adding the desktop styling. Currently, I can see your mobile view is broken. Kindly compare it with the figma template and fix it, please.
### Your design
![image](https://user-images.githubusercontent.com/94551484/229282998-0831ef0e-93b7-4b40-9c6b-6ba5d206f3f1.png)
![image](https://user-images.githubusercontent.com/94551484/229283168-41ee7249-ea2c-4961-84b3-22a566033f75.png)


### Figma
![image](https://user-images.githubusercontent.com/94551484/229283332-1df51405-7c0d-4f56-9771-afb7eef98ada.png)